apps/condo/domains/common/hooks/useSearch.ts

```diff
@@ -21,5 +21,5 @@ export const useSearch = <F> (loading: boolean): [string, (search: string) => vo
         searchChange(value)
     }
 
-    return [search, handleSearchChange]
+    return [search ?? searchValueFromQuery, handleSearchChange]
```

Comment:
we had some problems with compatibility for nullish coalescing operator. We need some tests and an agreement for using it. Better, to use 
```js
[search || searchValueFromQuery, handleSearchChange] 
```
for now

Reply:
@dkoviazin, fixed